### ISSUE TITLE
New alias management in settings complains about needing an alias on security tab

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#12672]
![image](https://user-images.githubusercontent.com/1190097/76280640-23790800-6258-11ea-9726-3cacba24039e.png)

The room has several aliases defined, including in the new way.

**Comment by bwindels** [comment]
Does your homeserver advertise support for the unstable feature `org.matrix.msc2432` in /versions?

**Comment by turt2live** [comment]
yes

**Comment by bwindels** [comment]
Hmm, are any of those aliases on your local homeserver?

**Comment by turt2live** [comment]
No, I'm just a regular user browsing the room settings.

**Comment by bwindels** [comment]
I mean, assuming your homeserver is `t2l.io`, are there any local aliases defined on `t2l.io`? Any user can add aliases fwiw.

**Comment by turt2live** [comment]
Nope, there's nothing.

**Comment by bwindels** [comment]
> Nope, there's nothing.

Yeah, that's why then. We only check the local aliases currently. I guess we could at least check the `alias` and `alt_aliases` in the canonical alias event, if any, too.
